core/domain/interaction_registry_test.py

```diff
@@ -87,6 +88,9 @@ class InteractionRegistryUnitTests(test_utils.GenericTestBase):
 
     def test_interaction_registry(self):
         """Do some sanity checks on the interaction registry."""
+        with patch(
+                'core.domain.interaction_registry.Registry._interactions', {}):
+            interaction_registry.Registry.get_all_interactions()
         self.assertEqual(
```

Comment:
Can you assert that the result from `interaction_registry.Registry.get_all_interactions()` is what you expect?

Reply:
Will do.